packages/beacon-node/src/chain/blocks/verifyBlock.ts

```diff
@@ -193,17 +206,23 @@ export async function verifyBlockStateTransition(
     switch (execResult.status) {
       case ExecutePayloadStatus.VALID:
         executionStatus = ExecutionStatus.Valid;
-        chain.forkChoice.validateLatestHash(execResult.latestValidHash, null);
+        chain.forkChoice.validateLatestHash({
```

Comment:
@g11tech my understanding is `latestValidHash` is the current execution payload which we have not imported into forkchoice, is it an issue?

Reply:
@tuyennhv its not an issue here, since the latestValidHash should be the parent of the current payload, as current payload should not have become part of the chain. But I will validate this.

Reply:
modified, just directly passing the parent's exechash now, so no matter what EL responds as LVH (block or its parent), we pass the correct one